doc/classes/BaseMaterial3D.xml

```diff
@@ -401,7 +403,7 @@
 			If [code]true[/code], triplanar mapping for [code]UV2[/code] is calculated in world space rather than object local space. See also [member uv2_triplanar].
 		</member>
 		<member name="vertex_color_is_srgb" type="bool" setter="set_flag" getter="get_flag" default="false">
-			If [code]true[/code], the model's vertex colors are processed as sRGB mode.
+			If [code]true[/code], the model's vertex colors are processed in sRGB color space. If [code]false[/code], the model's vertex colors are processed in linear color space. See also [member albedo_texture_force_srgb].
```

Comment:
This isn't quite correct. The color space in which they are processed has to do with what color space they are stored in (which is controlled by the user) 

``vertex_color_is_srgb`` flags the material to convert the vertex colors from srgb space into linear space. 

``vertex_color_is_srgb`` should be set whenever vertex colours are specified in srgb, otherwise it should be false.

The description under the ``FLAG_SRGB_VERTEX_COLOR`` constant is much more accurate